systemtest/src/main/java/io/strimzi/systemtest/utils/specific/CruiseControlUtils.java

```diff
@@ -29,8 +32,37 @@
     private static final String CRUISE_CONTROL_MODEL_TRAINING_SAMPLES_TOPIC = "strimzi.cruisecontrol.modeltrainingsamples"; // partitions 32 , rf - 2
     private static final String CRUISE_CONTROL_PARTITION_METRICS_SAMPLES_TOPIC = "strimzi.cruisecontrol.partitionmetricsamples"; // partitions 32 , rf - 2
 
+    private static final int CRUISE_CONTROL_DEFAULT_PORT = 9090;
+    private static final String CRUISE_CONTROL_ENDPOINT = "/kafkacruisecontrol/";
+
     private CruiseControlUtils() { }
 
+    public enum SupportedHttpMethods {
+        GET,
+        POST
+    }
+
+    public enum CruiseControlEndpoints {
+        STATE,
+        REBALANCE,
+        STOP_PROPOSAL_EXECUTION,
+        USER_TASKS
+    }
+
+    @SuppressWarnings("Regexp")
+    @SuppressFBWarnings("DM_CONVERT_CASE")
+    public static String call(SupportedHttpMethods method, CruiseControlEndpoints endpoint) {
```

Comment:
Maybe you can use better name for this method? Something like `callApi` ? But it's just a suggestion